src/zenml/zen_stores/migrations/versions/9971237fa937_artifact_visualizations.py

```diff
@@ -0,0 +1,61 @@
+"""Artifact Visualizations [9971237fa937].
+
+Revision ID: 9971237fa937
+Revises: 0.38.0
+Create Date: 2023-04-06 16:40:44.430701
+
+"""
+import sqlalchemy as sa
+import sqlmodel
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9971237fa937"
+down_revision = "0.38.0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    """Upgrade database schema and/or data, creating a new revision."""
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "artifact_visualization",
+        sa.Column("artifact_id", sqlmodel.sql.sqltypes.GUID(), nullable=False),
+        sa.Column("id", sqlmodel.sql.sqltypes.GUID(), nullable=False),
+        sa.Column("created", sa.DateTime(), nullable=False),
+        sa.Column("updated", sa.DateTime(), nullable=False),
+        sa.Column("type", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("uri", sa.TEXT(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["artifact_id"],
+            ["artifact.id"],
+            name="fk_artifact_visualization_artifact_id_artifact",
+            ondelete="CASCADE",
+        ),
+        sa.PrimaryKeyConstraint("id"),
+    )
+
+    with op.batch_alter_table("artifact", schema=None) as batch_op:
+        batch_op.alter_column(
+            "uri",
+            existing_type=sa.VARCHAR(),
+            type_=sa.TEXT(),
+            existing_nullable=False,
+        )
```

Comment:
Just curious about why this change of type was needed. Is this related to the field size ? IIRC there is no difference regarding the max size between TEXT and VARCHAR, so this change doesn't really do anything in that regard.

Reply:
On SQLite there is no difference since all strings are unlimited length varchars. On MySQL however, varchar always needs to have a max length and the default is 255. So `sa.VARCHAR()` means 255 chars max while `sa.TEXT` allows for up to 65535 chars.